pokeapp/src/components/Sidebar.js

```diff
@@ -258,8 +258,8 @@ const SideBar = ({
 
   return (
     <div className="sidebar">
-      {selectedPokemon.types ? sidebarInformation() : <p>Side Bar</p>}
-    </div>
+      {selectedPokemon.types ? sidebarInformation() : null}{" "}
```

Comment:
Do you still need the {""} looks like the same thing Garret had a bit back

Reply:
Didn't really mean to add that, just appeared after I added the comment, weird.